opencog/atomese/interpreter/Interpreter.cc

```diff
@@ -139,7 +140,52 @@ ValuePtr Interpreter::execute(const Type t, const ValueSeq& params)
 		result = logical_not( LinkValueCast(params[0]));
 		return ValuePtr(result);
 	}
-	return ValuePtr();
+
+	if (t == COND_LINK) {
+        ValueSeq result, conds, exps, default_exp, res2;
+        std::vector<double> result2, res;
```

Comment:
These variable names, especially `result2` and `res` could more telling, for instance `result2` could renamed `f_result` or such. Choosing names is an art, they should be as short and as information packed as possible.